examples/strategies/emacross.go

```diff
@@ -16,6 +16,9 @@ func (e CrossEMA) Timeframe() string {
 	return "1d"
 }
 
+// Stop can be used to close stuff of plot/log some strategy data
+func (e CrossEMA) Stop(df *model.Dataframe, broker exchange.Broker) {}
```

Comment:
The name `Stop` can be misunderstood (eg. stop order, custom stop, stop signal). 
May we can rename Init the function `Init` to `Finish` and keep `Start` and `Finish`
Do you have any other name suggestions?

Reply:
Agreed will change to finish. I called it stop to be consistent with the other stop functions scattered around the code.

Reply:
renamed to finish.